### DISCUSSION TITLE
All exceptions are marked as handled

### DISCUSSION THREAD

**Discussion by Jamesking56** [getsentry/sentry-symfony#669] [Q&A]
Hi,

We have an issue where Sentry is reporting (and alerting) us to exceptions that we are handling in the code. We don't care about those, we only want to know about unhandled exceptions.

So, I'm attempting to add a Sentry filter to filter out all of the handled exceptions by checking `$exception->getMechanism()->isHandled()`.

The problem I have though, is that every single exception thrown thinks it has been handled... regardless whether is has been caught or not.

Any ideas why this would be happening and how to stop it?

Cheers.

**Comment by cleptric** [comment]
We do expose a global `error_handler` that will report all uncaught exceptions to Sentry.
If you catch them yourself in your code, they should not be sent. Do you call `captureException()` somewhere in your code?

--

As the Symfony SDK uses `captureException()`, all Exceptions are marked as `handled: true` in the Sentry UI. We recently exposed the mechanism through the `EventHint` (https://github.com/getsentry/sentry-php/pull/1416), so people can manually overwrite this behavior. To clarify, this only changes the property of the Sentry event and has nothing to do with PHP.
